### PR TITLE
Whitebalance - Removed GUI Itcwb  from whitebalance and preferences 

### PR DESCRIPTION
**Pull request by Desmis**
I removed all GUI - sliders, frame,  checkbox used by Itcwb - in whitebalance.cc whitebalance.h and prefrences.cc.

The algorithm and settings are strictly the same as in the PR 
[https://github.com/Beep6581/RawTherapee/pull/6643](url)

jacques

### CODE REVIEW

File: rtdata/languages/default [hunk 1]
```diff
@@ -1528,6 +1528,17 @@ HISTORY_MSG_WAVSTREND;Strength soft
 HISTORY_MSG_WAVTHRDEN;Threshold local contrast
 HISTORY_MSG_WAVTHREND;Threshold local contrast
 HISTORY_MSG_WAVUSHAMET;Clarity method
+HISTORY_MSG_WBALANCE_OBSERVER10;Observer 10°
+HISTORY_MSG_WBITC_SAMPLING;Low sampling
```
Comment (Lawrence37) [inline rtdata/languages/default]: Entries for removed GUI components should be removed too so translators don't have extra work :)

**Comment by Lawrence37** [comment]
Aside from the default file, the code looks good to me. I'm busier than usual, so I won't have time to test until maybe this weekend.

**Comment by Desmis** [comment]
@Lawrence37 

I will push a change - remove all history_msg, Labels, Tooltips...from Itcwb

**Review by Lawrence37 — APPROVED** [review]

**Comment by Desmis** [comment]
@Lawrence37 

Thank you very much :)

If nobody objects, I will merge this PR tomorrow.

Jacques

**Pull request merged**